third_party/terraform/website/docs/d/datasource_google_sql_ca_certs.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Cloud SQL"
+layout: "google"
+page_title: "Google: google_sql_ca_certs"
+sidebar_current: "docs-google-datasource-sql-ca-certs"
+description: |-
+  Get all of the trusted Certificate Authorities (CAs) for the specified SQL database instance.
+---
+
+# google\_sql\_ca\_certs
+
+Get all of the trusted Certificate Authorities (CAs) for the specified SQL database instance. For more information see the
+[official documentation](https://cloud.google.com/sql/)
+and
+[API](https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/instances/listServerCas).
+
+
+## Example Usage
```

Comment:
Is there a terraform-relevant example of how this might be used, such as another resource that might interpolate on this value? If so, mind adding it as well?

Reply:
I am not able to find a TPG resource that could be interpolated into; typically this is used by applications to connect to the db via ssl. Therefore such a usage would be encapsulated as a terraform module via relevant outputs. 

I have extended my example a bit. Please see if it makes sense.

Reply:
Looks good!